Added SQLITE3_SUPPORT description [8920] (#179)
eProsima/Fast-RTPS-docs: docs/installation/configuration/cmake_options.rst

          option to `Debug`. Moreover, :class:`INTERNAL_DEBUG` is set to ``ON`` if |br|
          :class:`EPROSIMA_BUILD` is ``ON``.
        - ``OFF``
    *   - :class:`SQLITE3_SUPPORT`

>         - Builds  the |TRANSIENT_DURABILITY_QOS-api| and |PERSISTENT_DURABILITY_QOS-api| options for |br|

Reviewer: I'd say that it does not build the TRANSIENT and PERSISTENT QoS. It's more that it enables their use, since it builds the SQLite3 Plugin, which is the only one that the persistence service supports. I think you could actually change the PR to go to branch `feature/persistence`. That way, you can reference the persistence service sections

          option to `Debug`. Moreover, :class:`INTERNAL_DEBUG` is set to ``ON`` if |br|
          :class:`EPROSIMA_BUILD` is ``ON``.
        - ``OFF``
    *   - :class:`SQLITE3_SUPPORT`

>         - Builds the |SQLITE3_PLUGIN|. This enables the |TRANSIENT_DURABILITY_QOS-api| |br|
>           and |PERSISTENT_DURABILITY_QOS-api| options for the :ref:`durabilitykind`, which |br|
>           depend on the :ref:`persistence_service`.

Reviewer 2: ```suggestion
        - Builds the |SQLITE3_PLUGIN|, which enables the |TRANSIENT_DURABILITY_QOS-api| |br|
          and |PERSISTENT_DURABILITY_QOS-api| options for the :ref:`durabilitykind`and therefore the :ref:`persistence_service`.
```
